dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/engine/command/impl/SubmitDatasetForReviewCommand.java

```diff
@@ -58,15 +61,14 @@ public Dataset save(CommandContext ctxt)  {
         updateDatasetUser(ctxt);
 
         AuthenticatedUser requestor = getUser().isAuthenticated() ? (AuthenticatedUser) getUser() : null;
```

Comment:
Same as above.

Reply:
This is the original place that I've copied this logic